library/core/registry/src/testmod/java/org/quiltmc/qsl/registry/test/dynamic/RegistryLibDynamicRegistryTest.java

```diff
@@ -65,6 +72,23 @@ public void greetingsGetSynced(QuiltTestContext ctx) {
 		));
 	}
 
+	@GameTest(structureName = EMPTY_STRUCTURE)
+	public void greetingsTagGetLoaded(QuiltTestContext ctx) {
+		TagKey<Greetings> tagKey = TagKey.of(Greetings.REGISTRY_KEY, GREETING_TEST_TAG_ID);
+		Set<TagValues<Greetings>> tagValuesSet = TagRegistry.stream(Greetings.REGISTRY_KEY).collect(Collectors.toSet());
+		ctx.failIf(() -> ctx.assertTrue(tagValuesSet.isEmpty(), "tagValuesSet should always be populated with at least 1 object"));
+
+		ctx.succeedIf(() -> ctx.assertTrue(tagValuesSet.stream().anyMatch(tagValues -> {
+			Registry<Greetings> greetingsRegistry = ctx.getWorld().getRegistryManager().get(Greetings.REGISTRY_KEY);
```

Comment:
Yes, the original code had one of those too, I would change the other instance if I was me :p
```suggestion
			var greetingsRegistry = ctx.getWorld().getRegistryManager().get(Greetings.REGISTRY_KEY);
```